speculos/client.py

```diff
@@ -129,6 +129,11 @@ def finger_touch(self, x: int, y: int, delay: float = 0.5) -> None:
         with self.session.post(f"{self.api_url}/finger", json=data) as response:
             check_status_code(response, "/finger")
 
+    def ticker_ctl(self, action: str) -> None:
+        data = {"action": action}
```

Comment:
I prefer not adding it. It will be checked API side, and this is enough, client should not send invalid request, if they do, they can bear a useless HTTP request.